addon/webextension/background/main.js

```diff
@@ -40,8 +40,12 @@ window.main = (function () {
       });
   }
 
+  function opensMyShots(url) {
```

Comment:
That one "s" between openMyShots and opensMyShots makes quite a bit of difference – easy to misread?  `shouldOpenMyShots` ?